pkgs/by-name/ti/tinyfetch/package.nix

```
@@ -0,0 +1,34 @@
+{
+  stdenv,
+  gcc,
+  fetchFromGitHub,
+  lib,
+}:
+
+stdenv.mkDerivation {
+  pname = "tinyfetch";
+  version = "0.2";
+
+  src = fetchFromGitHub {
+    owner = "abrik1";
+    repo = "tinyfetch";
+    rev = "c23c22d8af6cf60f15e18c9859623e2faff03dbc";
+    hash = "sha256-nuC7Xtfg7rf6/IDhUKT2JZD9V0Hl/welNAFDJYJKwmA=";
+  };
+
+  buildInputs = [ gcc ];
+
+  buildPhase = ''
+    cd $src/src
+    mkdir -p $out/bin
+    gcc tinyfetch.c -o $out/bin/tinyfetch
+  '';
+
+  meta = {
+    description = "A simple fetch in C which is tiny and fast";
+    homepage = "https://github.com/abrik1/tinyfetch";
+    license = lib.licenses.mit;
+    mainProgram = "tinyfetch";
+    maintainers = with lib.maintainers; [ pagedMov ];
```

```suggestion
    maintainers = with lib.maintainers; [ pagedMov ];
    platforms = lib.platforms.unix;
```

Since it depends on some environment variables commonly found on unix systems